src/app/templates/scripts/_server.js

```diff
@@ -61,9 +61,9 @@ const tasks = {
       if (err) {
         reject(err.message);
       } else {
-        fs.writeFile(dests.css, result.css, (err) => {
-          if (err) {
-            reject(err.message);
+        fs.writeFile(dests.css, result.css, (errr) => {
```

Comment:
The linter complains about that because it's overriding the `err` from a previous scope.